cmd/krel/cmd/ff.go

```diff
@@ -128,7 +119,7 @@ func runFf(opts *ffOptions) error {
 	}
 
 	workingDir := filepath.Join(baseDir, branch)
-	log.Printf("working directory is %q", workingDir)
+	log.Printf("New working directory is %q", workingDir)
 
 	os.Setenv("GOPATH", workingDir)
 	log.Printf("GOPATH: %s", os.Getenv("GOPATH"))
```

Comment:
I think the whole GOPATH rewrite and symlink creation is not needed at all for the ff use-case. I tend to move it into the utils package because I'm sure we will need it later on. Should I do this?

Reply:
Sounds good. We won't need it anymore here, now that we're not worrying about running `hack/update-openapi-spec.sh`.